### ISSUE TITLE
plant reconstruct confusion

### ISSUE THREAD

**Issue by nconrad** [ModelSEED/PlantSEED#12]
It's been awhile since i've run reconstruct against against a workspace path and it may have been disabled.  Should we make this work or am I doing something wrong?

On `p3c.theseed.org/dev1/services` this request:

{"version":"1.1","method":"ProbModelSEED.ModelReconstruction","id":"7637585173361003","params":[{"genome":"/nconrad/plantseed/genomes/Zmays_Thi2"}]}

Throws:

{"id":"7637585173361003","error":{"name":"JSONRPCError","error":"Attribute (function) does not pass the type constraint because: Validation failed for 'Str' with value undef at constructor Bio::KBase::ObjectAPI::KBaseGenomes::Feature::new (defined at /disks/p3dev1/deployment/lib/Bio/KBase/ObjectAPI/KBaseGenomes/Feature.pm line 629) line 153\n\tBio::KBase::ObjectAPI::KBaseGenomes::Feature::new('Bio::KBase::ObjectAPI::KBaseGenomes::Feature', 'HASH(0xd7d4a00)') called at /disks/p3dev1/deployment/lib/Bio/KBase/ObjectAPI/BaseObject.pm line 793\n\tBio::KBase::ObjectAPI::BaseObject::_build_object('Bio::KBase::ObjectAPI::KBaseGenomes::Genome=HASH(0xc647ca0)', 'features', 'HASH(0xd8ef358)') called at /disks/p3dev1/deployment/lib/Bio/KBase/ObjectAPI/BaseObject.pm line 809\n\tBio::KBase::ObjectAPI::BaseObject::_build_all_objects('Bio::KBase::ObjectAPI::KBaseGenomes::Genome=HASH(0xc647ca0)', 'features') called at /disks/p3dev1/deployment/lib/Bio/KBase/ObjectAPI/KBaseGenomes/DB/Genome.pm line 292\n\tClass::MOP::Class:::around('CODE(0xfcde5e8)', 'Bio::KBase::ObjectAPI::KBaseGenomes::Genome=HASH(0xc647ca0)') called at /disks/patric-common/runtime/lib/perl5/site_perl/5.20.2/x86_64-linux/Class/MOP/Method/Wrapped.pm line 158\n\tClass::MOP::Method::Wrapped::__ANON__('Bio::KBase::ObjectAPI::KBaseGenomes::Genome=HAS  ...

**Comment by samseaver** [comment]
Yes, I got this error earlier, I mentioned it to you and I know what the problem is. I'm fixing it tonight. Essentially, if a function doesn't exist, it has to be an empty string, it cannot be 'undefined'.
